Fix package name

Whissi opened the pull request:
The previous package name "percona-toolkit" is not a valid package name for
ExtUtils::MakeMaker:

> $ perl Makefile.PL
> Checking if your kit is complete...
> Looks good
> Warning: NAME must be a package name
> [...]

Let us use the same package name (Percona::Toolkit) various scripts in bin/
are already using.

hydrapolic commented:
Can this be merged if we rebase it?

it-percona commented:
[![CLA assistant check](https://cla-assistant.percona.com:443/pull/badge/signed)](https://cla-assistant.percona.com:443/percona/percona-toolkit?pullRequest=84) <br/>All committers have signed the CLA.

svetasmirnova reviewed (approved):
Thank you for your contribution. Could you please sign CLA?

Whissi committed:
Fix package name

The previous package name "percona-toolkit" is not a valid package name for
ExtUtils::MakeMaker:

  $ perl Makefile.PL
  Checking if your kit is complete...
  Looks good
  Warning: NAME must be a package name
  [...]

Let us use the same package name (Percona::Toolkit) various scripts in bin/
are already using.

Whissi commented:
Rebased, CLA should be signed.
